cmd/root.go

```diff
@@ -225,12 +224,11 @@ func readLine(in io.Reader) (string, error) {
 // lineBreakCommand returns a new instance of a command to be used as a line break
 // in a command's help output.
 //
-// This is not a plain reference to flags.LineBreak,
+// This is not a plain reference to lineBreak,
```

Comment:
This comment could be updated to something like

> This returns a new instance of the lineBreakCommand every time to avoid data races in concurrent tests exercising commands.